chain/network/src/routing.rs

```diff
@@ -466,11 +491,17 @@ impl RoutingTable {
         (pings, pongs)
     }
 
-    pub fn info(&self) -> RoutingTableInfo {
-        let account_peers = self
-            .account_peers
-            .iter()
-            .map(|(key, value)| (key.clone(), value.peer_id.clone()))
+    pub fn info(&mut self) -> RoutingTableInfo {
+        // TODO(MarX): Fix: Allow SizedCache to iterate through values.
+        //  Iterate in reverse order to leave cache in the same state.
+        let accounts_id: Vec<_> = self.account_peers.key_order().cloned().collect();
```

Comment:
why not use `get_account_keys` here?

Reply:
true, I made an update directly on the `cached` library that allows to iterate by value. It already landed so I updated this.